js/utils/arrays.js

```diff
@@ -9,6 +9,16 @@ export const newArray = (module, b, isZero=false) => {
   return ptr
 }
 
+export const newArray0 = (module, len) => {
```

Comment:
Haha, ok … you really want this duplicated code i guess ;) please look at the `newArray` function, then you will see what i meant … this is already implemented there

Reply:
how do you expect `b.length` work when b doesn't exist ?